content/content-strategy/manage-content-requests/create-content/content-production-template/resources-cards.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: cards/cards
+cards:
+
+  - headline: Content Guide
+    text: This 'living' guide helps government teams create simple, clear and accessible digital content.
```

Comment:
This page links to Content **`guide`** in a slightly different context to the homepage. But should the description be the same as the homepage, or not? cc @Phend-117 and @SteveDTA  What do you think? Keep this blurb or make it the same as the approved one for the homepage?

Reply:
My 2 cents worth :  I think it's fine to have an abstract that suits the context.